### DISCUSSION TITLE
Not able to make badge for my custom logo

### DISCUSSION THREAD

**Discussion by ankanpy** [badges/shields#10219] [Q&A]
Trying to make a custom logo badge for my [logo](https://drive.google.com/file/d/1j5XAef_OEJ09ZufRvJesIgmvxBP6SA-N/view?usp=sharing)

but it shows --

**404 Badge Not Found**

Please help :)

**Comment by chris48s** [comment]
duplicate of https://github.com/badges/shields/issues/10220
